### PR TITLE
Replace methods with C# getters/setters

### LINKED ISSUE #6: Replace some methods by C# getters and setters
Now that MediaPipeUnityPlugin v0.8.2 for MediaPipe v0.8.9 has been ported one-to-one, we can start worrying about making changes to the framework so that it is nicer to use.

Some methods could be turned into C# properties for slightly better readability. Some examples are:
https://github.com/vignetteapp/MediaPipe.NET/blob/0eaa7242701ff16261934d9c8ccc95abe8cb5e57/Mediapipe.Net/Framework/Timestamp.cs#L42-L54
https://github.com/vignetteapp/MediaPipe.NET/blob/0eaa7242701ff16261934d9c8ccc95abe8cb5e57/Mediapipe.Net/Framework/CalculatorGraph.cs#L200-L215
https://github.com/vignetteapp/MediaPipe.NET/blob/0eaa7242701ff16261934d9c8ccc95abe8cb5e57/Mediapipe.Net/Framework/Format/ImageFrame.cs#L72-L114

### PR DESCRIPTION
**Pull request by adryzz**
Closes #6 

Should i do the `Ok()` as well?

### CODE REVIEW

File: Mediapipe.Net/Graphs/InstantMotionTracking/Anchor3dVector.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@
 namespace Mediapipe.Net.Graphs.InstantMotionTracking
 {
     [StructLayout(LayoutKind.Sequential)]
-    internal struct Anchor3dVector
+    internal struct Anchor3dVector : IDisposable
```
Comment (Speykious) [inline Mediapipe.Net/Graphs/InstantMotionTracking/Anchor3dVector.cs:13]: Why?

**Review by sr229 — APPROVED** [review]

**Review by Speykious — COMMENTED** [review]
Why?

File: Mediapipe.Net/Graphs/InstantMotionTracking/Anchor3dVector.cs (hunk 1 above)
Comment (adryzz) [reply Mediapipe.Net/Graphs/InstantMotionTracking/Anchor3dVector.cs:13]: it implemented `Dispose()`

Comment (Speykious) [reply Mediapipe.Net/Graphs/InstantMotionTracking/Anchor3dVector.cs:13]: Ah ok

**Pull request merged**